engine/src/flutter/lib/ui/painting.dart

```diff
@@ -1909,6 +1909,10 @@ enum PixelFormat {
   /// component, followed by: green, blue and alpha. Premultiplied alpha isn't
   /// used, matching [ImageByteFormat.rawExtendedRgba128].
   rgbaFloat32,
+
+  /// This is an unspecified pixel format that leaves it to Flutter's discretion
+  /// to choose the optimal pixel format.
+  optimal,
```

Comment:
How about `kDontCare`.

Whatever name you choose, perhaps also specify when someone might want to use this. Something like "Use this when the image is going to be sampled the application doesn't care about the exact format."

Reply:
Wait, this can't be right. This struct is given to decodeImageFromPixels. This isn't the target pixel format, its the incoming pixel format. How can Flutter divine this format?